ids/mapping.json

```diff
@@ -13,5 +13,6 @@
   "shihab": "shihabus",
   "swaraj": "swarajpure",
   "sweta": "karnsweta92",
-  "tejas": "tejasberi"
+  "tejas": "tejasberi",
+  "taksh": "takshch"
```

Comment:
The ids are supposed to be in alphabetical order, please shift your id line before tejas' id line

Reply:
Changes are made. Please pr review again.